libs/langchain/langchain/chat_models/fireworks.py

```diff
@@ -55,7 +67,7 @@ def _convert_delta_to_message_chunk(
     elif role or default_class == ChatMessageChunk:
         return ChatMessageChunk(content=content, role=role)
     else:
-        return default_class(content=content)
+        return default_class(type="", content=content)
```

Comment:
`type` should not be specified -- it's assigned by default depending on the message class

Reply:
I got a linter error says argument type is missing. Let me update it later to see if it passes the CI here.

Reply:
Have passed CI here without specifying value to type, so should be the linter I used was incorrect.